### DISCUSSION TITLE
Fedora: after SAML auth success get "Error: Invalid auth cookie"

### DISCUSSION THREAD

**Discussion by gitBrikski** [yuezk/GlobalProtect-openconnect#300] [General]
This wonderful product now seems to be tripping on updated GlobalProtect server.  On my Fedora 39 OS the current version (globalprotect-openconnect-2.0.0~beta5-1.1.x86_64) makes it through SAML authentication, but fails at the final (local?) cookie setting.  I've attached a log of STDERR reports, not sure where other logfiles live.

Thanks for 6 months of using a fantastic product.  I'll buy you another coffee or two if you can get this problem fixed.
[globalProtect-OpenConnect.log](https://github.com/yuezk/GlobalProtect-openconnect/files/14026662/globalProtect-OpenConnect.log)

**Comment by yuezk** [comment]
@gitBrikski You could try https://github.com/yuezk/GlobalProtect-openconnect/releases/tag/v2.0.0-beta7, which should fix this. But has not been published to COPR yet.

**Reply by gitBrikski** [reply]
Yep, that worked.  Enjoy the coffee...https://www.buymeacoffee.com/yuezk
